src/phoebus_utils/grid_utils.hpp

```diff
@@ -0,0 +1,58 @@
+// © 2022. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract
+// 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+// is operated by Triad National Security, LLC for the U.S.
+// Department of Energy/National Nuclear Security Administration. All
+// rights in the program are reserved by Triad National Security, LLC,
+// and the U.S. Department of Energy/National Nuclear Security
+// Administration. The Government is granted for itself and others
+// acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works,
+// distribute copies to the public, perform publicly and display
+// publicly, and to permit others to do so.
+
+#ifndef PHOEBUS_UTILS_GRID_UTILS_HPP_
+#define PHOEBUS_UTILS_GRID_UTILS_HPP_
+
+// Parthenon includes
+#include <coordinates/coordinates.hpp>
+#include <kokkos_abstraction.hpp>
+#include <parthenon/package.hpp>
+#include <utils/error_checking.hpp>
+
+namespace Coordinates {
+
+using namespace parthenon::package::prelude;
+using parthenon::Coordinates_t;
+
+template <int dir>
+KOKKOS_FORCEINLINE_FUNCTION Real GetXv(const int i, const Coordinates_t &coord);
+template <>
+KOKKOS_FORCEINLINE_FUNCTION Real GetXv<X1DIR>(const int i, const Coordinates_t &coord) {
+  return coord.x1v(i);
+}
+template <>
+KOKKOS_FORCEINLINE_FUNCTION Real GetXv<X2DIR>(const int i, const Coordinates_t &coord) {
+  return coord.x2v(i);
+}
+template <>
+KOKKOS_FORCEINLINE_FUNCTION Real GetXv<X3DIR>(const int i, const Coordinates_t &coord) {
+  return coord.x3v(i);
+}
```

Comment:
I use these in interpolation. Kind of annoying this functionality isn't in `parthenon`. Maybe it should be in a future change?

Reply:
lol fair enough. In Parthenon, I was imagining something like
```C++
coord.x1v(dir, i);
```
which wouldn't be too different than what's already there. But it's fine here too.